app/uk/gov/hmrc/uknwauthcheckerapi/models/AuthorisationRequest.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.uknwauthcheckerapi.models
+
+import play.api.libs.json._
+
+case class AuthorisationRequest(date: String, eoris: Array[String])
```

Comment:
Date should be LocalDate

Reply:
as per the openAPI spec its string so.

Reply:
It's a string, but it should be a localdate on our end. Otherwise anyone could send in garbage as the date. If it's a badly formed date, it'll 400.

Reply:
Replaced String with LocalDate.